src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_LoginPartial.OrgAuth.cshtml

```diff
@@ -0,0 +1,48 @@
+@using System.Security.Principal
+@*#if (IndividualB2CAuth)
+@using Microsoft.AspNetCore.Authentication.AzureADB2C.UI
+@using Microsoft.Extensions.Options
+@inject IOptionsMonitor<AzureADB2COptions> AzureADB2COptions
+
+@{ 
+    var options = AzureADB2COptions.Get(AzureADB2CDefaults.AuthenticationScheme);
+}
+    
+#endif *@
+
+@*#if (IndividualB2CAuth)
+@if (User.Identity.IsAuthenticated)
+{
+    <ul class="nav navbar-nav navbar-right">
+        @if (!string.IsNullOrEmpty(options.EditProfilePolicyId))
+        {
+            <li><a asp-area="AzureADB2C" asp-controller="Account" asp-action="EditProfile">Hello @User.Identity.Name!</a></li>
+        }
+        else
+        {
+            <li class="navbar-text">Hello @User.Identity.Name!</li>
+        }
+        <li><a asp-area="AzureADB2C" asp-controller="Account" asp-action="SignOut">Sign out</a></li>
+    </ul>
+}
+else
+{
+    <ul class="nav navbar-nav navbar-right">
+        <li><a asp-area="AzureADB2C" asp-controller="Account" asp-action="Signin">Sign in</a></li>
```

Comment:
`asp-action="SignIn"`? (capital `In` instead of `in`) Fix everywhere applicable. Seems like a nit, but users will see this.